aviary/subsystems/mass/gasp_based/design_load.py

```diff
@@ -655,7 +658,8 @@ class LiftCurveSlopeAtCruise(om.ExplicitComponent):
 
     def setup(self):
         add_aviary_input(self, Aircraft.Wing.ASPECT_RATIO, val=10.13)
-        add_aviary_input(self, Aircraft.Wing.SWEEP, val=0.436, units="rad")
+        add_aviary_input(self, Aircraft.Wing.SWEEP, val=25,
+                         units="deg")  # approx 0.436 rad
```

Comment:
we can still use degrees with set_val in other places where this component is used.

Reply:
Why is it better to leave it as is? Can you tell me how to do in `mass_builder.py` or `test_flops_mass_builder.py`?

Reply:
since it is always being used as radians in the calculations, it is better to have the component take in radians, instead of converting to degrees and then back to radians

Reply:
done.